docs-2.0/nebula-console.md

```diff
@@ -0,0 +1,178 @@
+# Nebula Console
+
+Nebula Console is a native CLI client for Nebula Graph, it can be used to connect a Nebula Graph cluster and execute queries, while supporting special commands to manage parameters, export CSV files, export DOT files, import test datasets, etc. This topic describes these special commands.
```

Comment:
```suggestion
Nebula Console is a native CLI client for Nebula Graph. It can be used to connect a Nebula Graph cluster and execute queries. It can also support special commands to manage parameters, export CSV files, export DOT files, import test datasets, etc. This topic describes these special commands.
```
when "while" is used in a sentence as a conjunction, it has two meanings. mostly we regard it as a contrast. and better use short sentences here.